src/main/java/cms/gov/madie/measure/utils/JsonUtil.java

```diff
@@ -224,12 +232,78 @@ public static List<TestCaseGroupPopulation> getTestCaseGroupPopulationsFromMeasu
               groupPopulations.add(groupPopulation);
             }
           }
+          // get Stratifications
+          JsonNode stratifications = group.get("stratifier");
+          if (stratifications != null) {
+            List<TestCaseStratificationValue> stratificationValues = new ArrayList<>();
+            AtomicInteger stratCount = new AtomicInteger(0);
+            stratifications.forEach(
+                (stratification) ->
+                    buildStratificationPopValues(
+                        group,
+                        stratificationValues,
+                        stratification,
+                        stratCount.getAndIncrement(),
+                        measurePopulationBasis));
+            groupPopulation.setStratificationValues(stratificationValues);
+          }
         }
       }
     }
     return groupPopulations;
   }
 
+  private static void buildStratificationPopValues(
+      JsonNode group,
```

Comment:
`group` doesn't seem to be used in this method, can it be removed?

Reply:
yep